iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -165,6 +165,9 @@ public HiveIcebergMetaHook(Configuration conf) {
 
   @Override
   public void preCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTable) {
+    if (hmsTable.isTemporary()) {
+      throw new UnsupportedOperationException("Creation of iceberg temporary tables is not supported.");
```

Comment:
I would change the order iceberg temporary table.  Also 
_"Creation of temporary iceberg tables is not supported"_

Reply:
Just a minor stuff.

Reply:
changed